main.go

```diff
@@ -48,25 +58,107 @@ func init() { // connect to Redis on init, and get highest group ID
 }
 
 func main() {
-    os.Setenv("apiPort", ":1323")
-
     e := echo.New()
 
     e.Use(mw.Logger())
     e.Use(mw.Recover())
 
     // Routes
-    e.Get("/groups", getAllGroups)
+    e.Get("/groups", GetGroups)
     e.Get("/groups/:id/gifs", getGroupGifs)
-    e.Post("/groups", createGroup)
+    e.Post("/groups", PostGroups)
     e.Post("/groups/:id/gifs", createGifInGroup)
 
-    e.Run(os.Getenv("apiPort"))
+    e.Run(os.Getenv("API_PORT"))
 }
 
-func getAllGroups(c *echo.Context) error {
-    var groups Groups
+func GetGroups(c *echo.Context) error {
+    res := &ResponseTemplate{}
+    res.Success = true
+    res.StatusCode = http.StatusOK
+    res.StatusText = http.StatusText(http.StatusOK)
+    res.ErrorCode = 0
+    res.ErrorText = "No Error"
```

Comment:
Decided I'd make a `func NewResponseTemplate` that will initialize these values to success. Will include in separate PR going up today.